bakalari_homework_tasklist/templates/_base.html

```diff
@@ -14,10 +14,13 @@
           href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5/css/solid.min.css"/>
     <link rel="stylesheet"
           href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5/css/fontawesome.min.css"/>
+    <link rel="stylesheet"
+	  href="https://fonts.googleapis.com/css?Family=Roboto:300display=swap">
```

Comment:
What does `display=swap` do? And isn't there supposed to be `&` before it? https://developers.google.com/fonts/docs/getting_started#use_font-display

Reply:
um yes there is supposed to be `&`. Its copied from google fonts idk what it means

Reply:
fixed in 0378468bd6bee88955f2abf55943616496a508cc